src/Microsoft.CodeAnalysis.BannedApiAnalyzers/BannedApiAnalyzers.Help.md

```diff
@@ -0,0 +1,50 @@
+How to use Microsoft.CodeAnalysis.BannedApiAnalyzers
```

Comment:
@genlu - as the md files are now autogenerated, I refactored this documentation that you recently added into a separate md file, which is linked from the analyzer help link URI (see changes to couple of files after this file)